pkg/reconciler/pipelinerun/cancel_test.go

```diff
@@ -95,6 +95,14 @@ func TestCancelPipelineRun(t *testing.T) {
 			{ObjectMeta: metav1.ObjectMeta{Name: "t1"}},
 			{ObjectMeta: metav1.ObjectMeta{Name: "t2"}},
 		},
+	}, {
+		name: "deprecated-state",
+		pipelineRun: &v1beta1.PipelineRun{
+			ObjectMeta: metav1.ObjectMeta{Name: "test-pipeline-run-cancelled"},
+			Spec: v1beta1.PipelineRunSpec{
+				Status: v1beta1.PipelineRunSpecStatusCancelledDeprecated,
```

Comment:
can we have a test with the non deprecated stopping and cancelling fields?

Reply:
This test was added to cover deprecated status name (PipelineRunCancelled). The already existing ones cover the new status name (Cancelled).